cmd/config.go

```diff
@@ -307,8 +307,9 @@ func (d *ConfigDuration) UnmarshalYAML(unmarshal func(interface{}) error) error
 // LogDescription contains the information needed to submit certificates
 // to a CT log and verify returned receipts
 type LogDescription struct {
-	URI string
-	Key string
+	URI    string
+	Key    string
+	MaxSPS int64
```

Comment:
Can you add a comment explaining what `MaxSPS` is? It might not be obvious to a casual reader that SPS -> "Submissions Per Second"